irods/__init__.py

```diff
@@ -3,6 +3,19 @@
 import logging
 import os
 
+def env_filename_from_keyword_args(kwargs):
+    try:
+        env_file = kwargs.pop('irods_env_file')
+    except KeyError:
+        try:
+            env_file = os.environ['IRODS_ENVIRONMENT_FILE']
+        except KeyError:
+            env_file = os.path.expanduser('~/.irods/irods_environment.json')
+    return env_file
+
+def derived_auth_filename(env_file):
```

Comment:
I don't really have a better suggestion for where to put these, BUT... is it normal to put these kinds of things in the `__init__.py` file?

Reply:
Was necessary due to import order.  irods is always imported before (ie in the process of) importing subordinate namespaces These being low level functions and subject to importation by other modules in subordinate namespaces (like `irods.test.helpers`) , and I didn't want to get into a cyclical import situation.

Reply:
Okay, makes sense to me.